style.css

```diff
@@ -571,18 +583,604 @@ footer h6 {
+    margin-top: 20px;
+    margin-bottom: 30px;
+  }
+
+  #list-of-project2::before {
+    content: url(./images/image-geometry2.png);
+    position: absolute;
+    top: -18px;
+    left: -124.8%;
+  }
+
+  .list-of-ptoject1 {
+    display: none;
+  }
+
+  #full-section-div2 {
+    display: block;
+    float: left;
+    width: 40%;
+    height: auto;
+    margin-left: 1%;
+    margin-top: 550px;
+    margin-right: 20%;
+  }
+
+  #full-section-div3 {
+    display: block;
+    width: 63.6%;
+    height: auto;
+    margin-left: 0;
+    margin-top: 910px;
+  }
+
+  #desktop-img1a {
+    display: block;
+    margin-left: 26.7%;
+    width: 200px;
+    height: 200px;
+    margin-top: 50px;
+  }
+
+  #desktop-img3 {
+    display: block;
+    width: 100px;
+    height: 100px;
+    margin-left: 863px;
+  }
+
+  #desktop-imgA {
+    display: block;
+    width: 100px;
+    height: 100px;
+    position: absolute;
+    left: 0;
+    top: 2738.5px;
+  }
+
+  #destop-img4 {
+    display: block;
+    width: 100px;
+    height: 100px;
+    margin-left: 1080px;
+    margin-top: -200px;
+  }
+
+  #img-geo {
+    display: block;
+    position: absolute;
+    left: 12px;
+    top: 2000px;
+    width: 200px;
+    height: 200px;
+  }
+
+  #full-section-div4 {
+    width: 32%;
+    margin-left: 963.6px;
+    margin-top: 0;
+  }
+
+  #full-section-div4::before {
+    content: url(./images/Shape.png);
+    position: absolute;
+    right: 220px;
+    margin-right: 10.5px;
+    top: 999px;
+    border: #1a2236 2px solid;
+  }
+
+  #full-section-div6 {
+    display: none;
+  }
+
+  #list-of-project3 {
+    display: none;
+  }
+
+  #list-of-project4 {
+    display: none;
+  }
+
+  #list-of-project5 {
+    display: none;
+  }
+
+  .project-list-1 {
+    padding: 20px 50px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 12px;
+  }
+
+  #list-of-project6 {
+    display: none;
+  }
+
+  .project-list-2 {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 120px;
+    padding: 20px 20px;
+    text-align: center;
+    margin: 0.2em;
+    font-size: 12px;
+    background-color: #98dede;
+    color: #1a2236;
+  }
+
+  .project-list-3 {
+    display: none;
+  }
+
+  .project-list-4 {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0.2em;
+    width: 54px;
+    height: 12px;
+    padding: 20px 40px;
+    font-size: 12px;
+    text-align: center;
+    background-color: #98dede;
+    color: #1a2236;
+  }
+
+  .contact-form {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+    flex-direction: column;
+    width: 100%;
+    height: auto;
+    place-items: center;
+    position: relative;
+    top: 16000;
+  }
+
+  .contact-form p {
+    width: 40%;
+    height: 100px;
+    color: #1a2236;
+    align-self: center;
+    text-align: center;
+    margin: 10px 30%;
+    font-size: 16px;
+    font-weight: 400;
+  }
```

Comment:
- [ ] Your contact form text is overlapping the form in some screen sizes. Kindly fix it for better readability.
![image](https://user-images.githubusercontent.com/61919617/172936328-3c1b71fe-f376-44b2-af0c-c03b7ca0e860.png)

Reply:
done